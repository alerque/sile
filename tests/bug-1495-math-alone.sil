\begin[papersize=a7]{document}
    \script[src=packages/math]
    \math{x}\cr\math{y}
\end{document}
