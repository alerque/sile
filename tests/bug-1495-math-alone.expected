Set paper size 	209.7637818	297.6377985
Begin page
Mx 	33.1308
My 	19.2719
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2735 w=5.4100	(nil)
Mx 	10.4882
My 	31.2719
T	2736 w=5.4000	(nil)
Mx 	102.5357
My 	279.9059
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
